styles.css

```
@@ -0,0 +1,162 @@
+
+
+body {
+  background-color:hsl(38.7,40.4%,90.8%);
+  font-family: 'Kanit', sans-serif;
+  line-height: 1.5;
+  margin: 24px;
+}
+
+h1 {
+  font-family: 'Architects Daughter', cursive;
+  font-size: 52px;
+  text-transform: capitalize;
+  font-weight: bold;
+  text-align: center;
+  color: hsl(20.4,55.6%,51.4%); 
+}
+
+h2{
+   font-family: 'Architects Daughter', cursive;
+   font-size: 40px;
+  font-weight: bold;
+  color: hsl(39.6,98.7%,70.8%); 
+  
+  
+}
+
+h3{
+  font-size:30px; 
+}
+
+h4{
+  font-size:24px;
+}
+
+p{
+ font-size:16px; 
+  margin-left:28px;
+  margin-right:28px;
+}
+
+.u-list-reset {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+.u-list-inline li {
+  display: inline-block;
+}
+
+
+
+.talks {
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+}
+
+.talk1,.talk2,.talk3{
+  padding-bottom:70px;
+  margin-left:0;
+}
+
+ 
+.form {
+  list-style: none;
+  margin: 0;
+  padding-top:0;
+  padding-left: 0;
+  padding-bottom:30px;
+  line-height: 2;
+}
+
+  .button-reset {
+  background: transparent;
+  border: 0;
+  cursor: pointer;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: inherit;
+  padding: 3px;
+  text-align: left;  
+  background-color:hsl(20.4,55.6%,51.4%) ;
+  border: 2px solid hsl(282, 36%, 16%);
+  color: white;
+}
+
+.button:hover,
+.button:focus {
+ color:hsl(39.6,98.7%,70.8%); ;
+}
+
+
+nav {
+  color: ;
+  font-size: 24px;
+  text-align: center;
+}
+
+nav a {
+  color: hsl(39.6,98.7%v,70.8%);
```

This value is broken and, therefore, doesn't work in the browser:
```suggestion
  color: hsl(39.6,98.7%,70.8%);
```